modules/apigee_edge_teams/src/Entity/TeamInvitation.php

```diff
@@ -190,7 +190,9 @@ public function setStatus(int $status): TeamInvitationInterface {
    * {@inheritdoc}
    */
   public function getRecipient(): ?string {
-    return $this->get('recipient')->value;
+    $recipientEmail = $this->get('recipient')->value;;
+    $user = user_load_by_mail($recipientEmail);
```

Comment:
This approach looks quite expensive and if the email server case sensitive then it could also lead to problems.

Case sensitivity is a delicate topic anyway... 

https://www.googlecloudcommunity.com/gc/Apigee/Apigee-Edge-to-Apigee-X-migration-INVALID-ARGUMENT-email-address/m-p/641748/highlight/true#M77284